### PR TITLE
Curve converter documentation

### PR DESCRIPTION
**Pull request by SamuilsRulovs**

### CODE REVIEW

**Comment by SamuilsRulovs** [comment]
Unfortunately it was not jet possible to create a nested numbered list. I am looking for solution to this.

**Comment by pjanck** [comment]
Nested numbering:

1. item
    1. with additional spaces
        1. you can achieve
        2. different levels
              1. of numbering
        3. got it?

File: Documentation/markdown/IfcGeometryConverter/CurveConverterDocumenetation.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# CurveConverter.h Documentation 
+
+This short guide shows you which functions are included in CurveConverter.h
```
Comment (pjanck) [inline Documentation/markdown/IfcGeometryConverter/CurveConverterDocumenetation.md:3]: Please include also the full path to the file

File: Documentation/markdown/IfcGeometryConverter/CurveConverterDocumenetation.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# CurveConverter.h Documentation
```
Comment (pjanck) [inline Documentation/markdown/IfcGeometryConverter/CurveConverterDocumenetation.md:1]: filename typo?

**Review by pjanck — CHANGES_REQUESTED** [review]

**Comment by SamuilsRulovs** [comment]
I guess this pull request is not necessary anymore, since all information is in the Doxymentation

**Pull request closed without merging**